api/src/config.js

```diff
@@ -88,16 +88,19 @@ const loadTranslations = () => {
   const options = { key: ['translations', true], include_docs: true };
   return db.medic
     .query('medic-client/doc_by_type', options)
+    .catch(err => {
+      logger.error('Error loading translations - starting up anyway: %o', err);
+    })
     .then(result => {
+      if (!result) {
+        return;
+      }
```

Comment:
I put the catch there in the first place (earlier in `3.5` when switching from callbacks) not realizing that previously API would crash when translations were malformed.

Reply:
Yea, and potentially creating untranslated sms messages.